code/src/main/java/com/onlinecontacttracing/servlets/authentication/CheckForApiAuthorizationServlet.java

```diff
@@ -113,18 +121,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   /**
```

Comment:
Update the docs too. Probably remove them since it is private now.

Reply:
done